### PR TITLE
adding google style compile without run on project

### PR DESCRIPTION
**Pull request by lucapreziati**
Added google style formatted during maven install phase.

### CODE REVIEW

**Comment by malafeev** [comment]
test is failing

**Comment by malafeev** [comment]
[ERROR] Failed to execute goal net.revelc.code.formatter:formatter-maven-plugin:2.10.0:format (default) on project opentracing-jms-common: Cannot find config file [/home/travis/build/opentracing-contrib/java-jms\google-style-formatter.xml] -> [Help 1]

File: pom.xml [hunk 1]
```diff
@@ -222,6 +223,22 @@
           <forkCount>1</forkCount>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>net.revelc.code.formatter</groupId>
+        <artifactId>formatter-maven-plugin</artifactId>
+        <version>2.10.0</version>
+        <configuration>
+          <encoding>UTF-8</encoding>
+          <configFile>${maven.multiModuleProjectDirectory}\google-style-formatter.xml</configFile>
```
Comment (malafeev) [inline pom.xml]: should be `/` instead of `\`

Comment (lucapreziati) [reply pom.xml]: I fix in few minutes

**Pull request merged**